app/src/components/filterEntities/entityItemStartTime.jsx

```diff
@@ -39,12 +33,9 @@ const presets = [
     value: {
       start: moment()
         .startOf('day')
-        .subtract(1, 'weeks')
+        .subtract(6, 'days')
```

Comment:
Why 6 days?

Reply:
Expected:
"filter.btw.start_time":"-1440;362;+0300" ('1440' is number of minutes of the previous day,'362' is number of minutes till the current moment of filtering and +0300 is the user's timezone)
Launches with start time in that range are shown. It was for Last 2 days. So for Last 7 days we should send "-1440*6;number of minutes till the current moment of filtering;+0300"